packages/liquidator/index.js

```diff
@@ -122,6 +126,13 @@ async function run({
       withdrawLiveness
     };
 
+    // Add block window into `liquidatorConfig`
+    let overriddenLiquidatorConfig = {
```

Comment:
does this need to be a new variable? could you not just re-asign `liquidatorConfig`?
```
liquidatorConfig = {...liquidatorConfig, startingBlock, endingBlock };
```

The only reason I bring this up is that it is more consistent with how I was trying to override the liquidator config in the perpetual generalization.

Reply:
Good call, I originally had it set this way but ran into an error because I redeclared `liquidatorConfig` with a `let` statement.